parser-preset.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  parserOpts: {
+    headerPattern: /^(\w*)-(\w*) - (.*)$/,
+    headerCorrespondence: ['type', 'id', 'subject'],
```

Comment:
[?] What is the purpose of these headerCorrespondence? I cannot see them in the code

Reply:
"headerCorrespondence" is for customizing object response

you have access to it in a custom rule ```({type, id})``` or all object ```(parser)```

'id' is a custom filed for a number

Reply:
every '-' and ':' in "headerPattern" is a comma in "headerCorrespondence" array

Reply:
Ok I see, so this is representation of groups in this regex